sampleapp/build.gradle

```diff
@@ -0,0 +1,50 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion rootProject.ext.targetSdkVersion
+
+    defaultConfig {
+        applicationId "com.zendesk.sampleapp"
+        minSdkVersion rootProject.ext.minSdkVersion
+        targetSdkVersion rootProject.ext.targetSdkVersion
+        versionCode 1
+        versionName "1.0"
+
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+
+}
+
+repositories {
+    google()
+    jcenter()
+    mavenCentral()
+    maven {
+        url  "http://dl.bintray.com/outbound/maven"
+    }
+}
+
+dependencies {
+    ext.shouldUseLocalDependency = true
+
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
```

Comment:
This line is auto-generated when you create a project/module but kind of flies in the face of usual repo-based Gradle dependency management, and usually isn't needed. Do we have any local jars in the module? Can we remove this line?